src/components/sbb-button/sbb-button.tsx

```diff
@@ -1,27 +1,26 @@
 import { Component, Event, EventEmitter, h, Prop } from '@stencil/core';
 import { InterfaceButtonAttributes } from './sbb-button.custom';
 
-/**
- * @slot unnamed - Slot to render svg icon. You must pass an svg-element.
- */
-
 @Component({
   shadow: true,
   styleUrl: 'sbb-button.scss',
   tag: 'sbb-button',
 })
 export class SbbButton {
-  /** Label text to show on the button */
-  @Prop() public label? = '';
+  /** set as icon-only, no label, no text */
+  @Prop() public iconOnly = false;
 
   /** Variant of the button, like primary, secondary etc. */
   @Prop() public variant?: InterfaceButtonAttributes['variant'] = 'primary';
 
   /** Size variant, either l or m. */
   @Prop() public size?: InterfaceButtonAttributes['size'] = 'l';
 
+  /** The href value you want to link to */
+  @Prop() public href?: string;
+
   /**
-   * Set this property to true if you want only a visual represenation of a
+   * Set this property to true if you want only a visual representation of a
    * button, but no interaction (a div instead of a button will be rendered).
    */
   @Prop() public visualButtonOnly?: boolean;
```

Comment:
visualButtonOnly should be removed and replaced with the same functionality as the one `sbb-link` uses.

Reply:
I left the visualButtonOnly prop, because the host-element check does not work correctly with shadow-DOMs.

Reply:
replaced by static as discussed and as it is in link